Header header

float64 easting
float64 northing

int32 context_score
